pkg/agent/cniserver/interface_configuration_windows.go

```diff
@@ -51,7 +51,6 @@ var (
 	getNamespaceEndpointIDsFunc     = hcn.GetNamespaceEndpointIds
 	hotAttachEndpointFunc           = hcsshim.HotAttachEndpoint
```

Comment:
`hotAttachEndpointFunc` is supposed to be called with Docker runtime. So we shall remove it after not supporting docker.